__tests__/components/NetworkSwitcher.test.js

```diff
@@ -6,7 +6,12 @@ import NetworkSwitcher from '../../src/app/components/NetworkSwitcher'
 
 const setup = () => {
   const props = {
-    network: { name: 'MainNet' },
+    networkId: 0,
```

Comment:
Can you explain the difference between the networkId and the networks map below it?

Reply:
network/networkId is the currently selected network. Networks is a listing of all available networks.

Reply:
Right but I'm wondering how they map to each other, or if they do at all.

Reply:
state.network.id is the id for accessing the config object states.config.networks. you think we should just add it to state.config.currentNetworkId or something like that?